server/README.md

```diff
@@ -30,3 +32,4 @@ FIREBASE_CLIENT_CERT_URL="https://www.googleapis.com/robot/v1/metadata/x509/fire
 BUILD="development" # set this in your testing enviroment as is here
                     # use api key '1234' for testing
 ```
+>>>>>>> 76645babd0b9a909616b3f3d58ebb033b7399e74
```

Comment:
After a merge conflict is resolved, remove the markers that git adds. It's lines 14, 15, and 35 in this case.